kubernetes/.swagger-codegen/VERSION

```diff
@@ -0,0 +1 @@
+2.3.0-SNAPSHOT
```

Comment:
(btw this is not blocker for this PR, will move the discussion to https://github.com/kubernetes-client/gen/issues/5)